Editor/EditorCore/ProBuilderToolManager.cs

```diff
@@ -95,8 +95,6 @@ public void Dispose()
             m_IsDisposed = true;
             GC.SuppressFinalize(this);
 
-            SetSelectMode(SelectMode.Object);
```

Comment:
Not sure this should be removed.
It ensures that Select mode (and in more recent version, the EditorTools Context) are restored when closing the ProBuilder editor window, insuring to restore GameObject context for the tools.
It sure set the selectmode to object when entering playmode but this is then updating by the probuilder editor.
What do you think @karl-  ?

Reply:
Yes I think this needs to remain here.